docs/osf-trivia-list/dasharo.md

```diff
@@ -124,3 +124,33 @@ We want to work on changing the above paradigms or at least improve the
 state-of-the-art relation in the community to the level where threats will have
 a reasonably small impact. We believe that open-source firmware is a critical
 tool, which should be used consciously to ensure privacy and liberty.
+
+## What is open-source firmware distribution?
+
+Dasharo is 3mdeb's firmware distribution and all its components are
+open-source. We provide releases in binary form. As you know in most cases on
+x86 for firmware to be useful it has to cooperate with closed blobs. In all
+binary releases we are making sure we provide information where all components
+are coming from.
+
+Dasharo works without blobs on platforms that allow that. When we saying
+Dasharo open-source firmware distribution we mean code that is delivered by
+3mdeb that is open-source. We have no influence on the code provided by 3rd
+parties (e.g. FSP, ME, GbE etc.).
+
+In coreboot community there was some controversy about calling Dasharo
+open-source firmware distribution (for details please check [gerrit review][gerrit-review].
```

Comment:
@miczyg1 missing URL?

Reply:
Also, closing bracket is missing

Reply:
URL fix was applied thanks to rebasing on top of dasharo_port branch.

Bracket fixed here: https://github.com/Dasharo/docs/pull/503/commits/bbba94259fb408b3b6b6fe150a6e92cf8e1ce244